tests/ExporterTest.php

```diff
@@ -613,4 +618,17 @@ protected function getRegistryMock($methods = null)
 
         return $mock;
     }
-}
\ No newline at end of file
+
+    /**
+     * @param array|null $methods
+     * @return Client|MockObject
+     * @throws \ReflectionException
+     */
+    protected function getHttpClientMock($methods = null)
```

Comment:
Instead of copying an already existing function from one test to another consider creating a `ClientHelper` trait in the `tests` folder with this method being public.